homeassistant/components/air_quality/waqi.py

```diff
@@ -137,7 +132,7 @@ def unit_of_measurement(self):
         return 'AQI'
 
     @property
-    def device_state_attributes(self):
+    def state_attributes(self):
```

Comment:
this should be device_state_attributes. `state_attributes` is reserved for entity component base classes.

Reply:
Fixed.